tensorrt_llm/_torch/attention_backend/flashinfer.py

```diff
@@ -122,7 +122,7 @@ def __post_init__(self) -> None:
         super().__post_init__()
 
         if self.workspace_buffer is None:
-            self.workspace_buffer = torch.empty(256 * 1024 * 1024,
+            self.workspace_buffer = torch.empty(512 * 1024 * 1024,
```

Comment:
Note: even though flashinfer recommends only 128MB, warmup will fail without this change. Will add a comment on next iteration